src/app/core/store/collections/collections.state.ts

```diff
@@ -46,4 +46,8 @@ export const selectCollectionsByQuery = createSelector(selectCollectionsDictiona
   return !query || query.collectionIds.length === 0 ? Object.values(collections) : query.collectionIds.map(id => collections[id]);
 });
 
+export const selectCollection = createSelector(selectCollectionsDictionary, selectWorkspace, (collections, workspace) => {
```

Comment:
`selectCollectionFromWorkspace` would be more clear in my opinion.

Reply:
Good idea.